tests/template_cassdc_test.go

```diff
@@ -0,0 +1,117 @@
+package tests
```

Comment:
Is this test the same as the one from #99? If so does it make sense to remove it from this PR?

Reply:
Yes, since it's a branch of that PR. A rebase will remove these commits.